### ISSUE TITLE
Warning on Sectioning content without headings

### ISSUE THREAD

**Issue by SaekiTominaga** [validator/validator#1448]
The following HTML will display warnings about headings only for `<article>` and `<section>`.

```html
<!DOCTYPE html><html lang=""><title>Test</title>

<article>article</article>
<aside>aside</aside>
<nav>nav</nav>
<section>section</section>
```

The HTML specification does not state that any element must have a heading.
However, only `<section>` has the statement <q>[typically with a heading](https://html.spec.whatwg.org/multipage/sections.html#the-section-element)</q>.

References to headings in Sectioning content and checker results when headings are omitted are as follows:

| Element | Role | HTML Spec | WAI-ARIA 1.2 (label) | WAI-ARIA 1.2 (Accessible Name) | Nu Html Checker |
| --- | --- | --- | --- | --- | --- |
| `<article>` | `article` | ― | ― | ― | Warning |
| `<aside>` | `complementary` | ― | ― | ― | ― |
| `<nav>` | `navigation` | ― | ― | ― | ― |
| `<section>` | `region` | <q>typically with a heading</q> | **MUST** | Required | Warning |

Why is Nu Html Checker warning only for `<article>` and `<section>`?

**Comment by sideshowbarker** [comment]
Closing, as I don’t personally plan on working on any behavior change for this. If anybody ends up stepping forward to say they’re actually going to work on it, we could re-open it at that time.
